src/components/FlatmapVuer.vue

```diff
@@ -843,13 +980,28 @@ export default {
           ) {
             this.checkAndCreatePopups(payload)
           }
+          /**
```

Comment:
I would switch this to:

```
          /**
           * The event emitted from the mouse event callbacks that come from flatmap-viewer. This event then 
           *  goes on to provide the information for opening datasets, scaffolds, and searches on the sidebar
           * @arg payload
           */
   ```

Reply:
@akhuoa

Reply:
I will change it to

/**
* The event emitted from the mouse event callbacks that come from flatmap-viewer. The payload
* argument provide an object with information on the featured where the mouse event takes place.
*@arg payload
*/

I will leave out the reset as they are part not part of this library.

Reply:
Thanks for the review, @alan-wu and @Tehsurfer. The "resource-selected" event description is updated in 50867aa.